sdk/Trace/Span.php

```diff
@@ -47,6 +47,44 @@ class Span implements API\Span, ReadableSpan
     /** @var ?SpanProcessor */
     private $spanProcessor;
 
+    /**
+     * @todo Implement this in the API layer
+     */
+    public static function fromContext(Context $context): API\Span
+    {
+        if ($span = $context->get(SpanContextKey::instance())) {
+            return $span;
+        }
+
+        return self::getInvalid();
+    }
+
+    /**
+     * @todo Implement this in the API layer
+     */
+    public static function getCurrent(): API\Span
+    {
+        return self::fromContext(Context::getCurrent());
+    }
+
+    public static function getInvalid(): NoopSpan
+    {
+        if (null === self::$invalidSpan) {
+            self::$invalidSpan = new NoopSpan();
+        }
+
+        return self::$invalidSpan;
+    }
```

Comment:
Kinda annoying needing to have a static var just for this, but is needed so it returns the same instance given we cant just define it as a PHP const. Maybe could via `define`, but at that point would be easier to have a dedicated type and just have this method delegate to that.